examples/docking-protein-ligand-shape/data/ligand.param

```diff
@@ -0,0 +1,214 @@
+! Put in this file your ligand parameters if needed
```

Comment:
Naive question. Can't these parameters be part of the HADDOCK3 supported ligands?

Reply:
Nope because we might every time deal with different ligands. If these were co-factors it would be different.
And good to show in an example how to provide the param/topo files